wiki/Game_Modes/External_Ports/T-Aiko!/Interface/en.md

```diff
@@ -0,0 +1,125 @@
+<!-- If possible, use a script to do this, mate. Pardon the terrible naming sense. ~RaikireHiuduo -->
+
+<!-- Reference: [{image}]: {link}/{image} "{altText}" -->
+<!-- Content  : ![{altText}][{image}] \n\n _{altText}_ -->
+
+<!-- Original page -->
+[T-Aiko!]: /wiki/Game_Modes/External_Ports/T-Aiko! "T-Aiko! main page"
+
+<!-- Normal -->
+[T2-MM.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-MM.jpg "Main Menu"
+[T2-LogIn.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-LogIn.jpg "Log-in pop-up"
+[T2-SignUp.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-SignUp.jpg "Sign Up pop-up"
+[T2-SS.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-SS.jpg "Song Selection (Solo)"
+[T2-SSblank.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-SSblank.jpg "Song Selection (blank)"
+[T2-SSDL.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-SSDL.jpg "Download stream"
+[T2-DELETE.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-DELETE.jpg "Delete screen"
+[T2-Gameplay.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-Gameplay.jpg "Gameplay example"
+[T2-Spinner.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-Spinner.jpg "Spinner"
+[T2-Paused.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-Paused.jpg "Paused screen"
+[T2-Failed.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-Failed.jpg "Failed screen"
+[T2-Grade.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-Grade.jpg "Grade screen"
+[T2-Guest.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-Guest.jpg "Main Menu (Guest)"
+
+<!-- Online -->
+[T2-ONLINE.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-ONLINE.jpg "Online screen"
+[T2-RANKOWN.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-RANKOWN.jpg "Own Ranking"
+[T2-RANKTOP.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-RANKTOP.jpg "Top Ranking"
+[T2-SSO.png]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-SSO.png "[Daily] Competitive Mode Song Selection"
+[T2-SSOB.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-SSOB.jpg "Competitive Mode Song Selection (blank)"
+[T2-CMDL.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-CMDL.jpg "Competitive Mode Download stream"
+[T2-CMINFO.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-CMINFO.jpg "Competitive Mode Information"
+[T2-CMGrade.jpg]: /wiki/Game_Modes/External_Ports/T-Aiko!/img/T2-CMGrade.jpg "Competitive Mode Grade screen"
+
+<!-- Title -->
+# T-Aiko v2 Interface
+
+**[Click here to return to T-Aiko! main page.][T-Aiko!]**
```

Comment:
link derp...
use `**[Click here to return to T-Aiko! main page.][../]**`

---
`../` means "go back one folder"
`T-Aiko!` means "to into `T-Aiko!` which will give you `/wiki/T-Aiko!/Interface/T-Aiko!`